feat(cliWrapper): wrap the cucumber CLI to keep the use of custom options (#69)
ekino/veggies: doc/README.tpl.md

[github-star]: https://github.com/ekino/veggies/stargazers
[twitter]: https://twitter.com/intent/tweet?text=Check%20out%20veggies!%20https://github.com/ekino/veggies%20%F0%9F%91%8D
[twitter-badge]: https://img.shields.io/twitter/url/https/github.com/ekino/veggies.svg?style=social

## CLI

Veggies provides a simple CLI allowing the use of custom options to configure the feature it provides.

> To make use of it, you can either use one of those commands:

Reviewer: ```suggestion
To make use of it, you can use either of these commands:
```

Author: Done.

```shell
$ yarn veggies

$ npx veggies

$ ./node_modules/.bin/veggies.js
```

The available options are:

| Option                      | Description                                                                                |
|-----------------------------|--------------------------------------------------------------------------------------------|

> | --cleanSnapshots            | remove unused snapshots (not recommended while matching tags)                              |

Reviewer: ```suggestion
| --cleanSnapshots            | removes unused snapshots (not recommended while matching tags)                              |
```

Author: Done.

> | -u, --updateSnapshots       | update current snapshots if required                                                       |

Reviewer: ```suggestion
| -u, --updateSnapshots       | updates current snapshots if required                                                       |
```

Author: Done.
